packages/cli/command_help.pony

```diff
@@ -0,0 +1,196 @@
+
+primitive Help
+
+  fun general(cs: CommandSpec box): CommandHelp
+  =>
+    CommandHelp._create(None, cs)
+
+  fun for_command(cs: CommandSpec box, argv: Array[String] box):
+    (CommandHelp | SyntaxError)
+  =>
+    let ch = CommandHelp._create(None, cs)
+    _parse(cs, ch, argv)
```

Comment:
I did want to keep the help support separate from the core CommandSpec. Making Help private would work for me.